azure-pipelines.yml

```diff
@@ -69,6 +67,24 @@ jobs:
           .\workload-install.ps1 -DotnetTargetVersionBand $(TOOLKIT_NET6_VERSION)
         displayName: Install tizen workload
 
+      # build sample
+      - task: CmdLine@2
+        displayName: 'Build Community Toolkit Sample'
+        inputs:
+          script: |
+            dotnet build $(PathToCommunityToolkitSampleCsproj) -c Release
+
+  - job: build_library
+    displayName: Build Library
+    strategy:
+      matrix:
+        'Windows':
+          image: 'windows-latest'
+        'macOS':
+          image: 'macos-12'
+    pool:
+      vmImage: $(image)
+    steps:
```

Comment:
We need to add a step for install tizen workload, because tizen workload not providing from visual studio

Reply:
Thanks! Fixed ✅ 

I missed it when resolving the merge conflicts 🙈